spec-views/after.rb

```diff
@@ -1,14 +0,0 @@
-require "yaml"
```

Comment:
we did at the beginning when we had limited knowledge about rspec, but it was not really used. They should've been removed a long time ago.